### ISSUE TITLE
[LINUX] Unable to load shared library 'webview' or one of its dependencies. libwebview: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by darkguy2008** [webview/webview_csharp#9]
Hi,

Just trying the example found in the Readme. Compiles correctly but then it says:
```
Exception has occurred: CLR/System.DllNotFoundException
An unhandled exception of type 'System.DllNotFoundException' occurred in SharpWebview.dll: 'Unable to load shared library 'webview' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: libwebview: cannot open shared object file: No such file or directory'
   at SharpWebview.Bindings.webview_create(Int32 debug, IntPtr window)
   at SharpWebview.Webview..ctor(Boolean debug, Boolean interceptExternalLinks)
   at App.Program.Main(String[] args) in Program.cs:line 13
```
How do I make this work then? I had expected that the nuget package included bindings for Win/Linux/OSX?

The bindings and libs seem to exist though. I'm stumped.
![image](https://user-images.githubusercontent.com/6304136/96408389-a411c680-11a0-11eb-9f2f-dbb251452dd5.png)

**Comment by geaz** [comment]
Hi! Yes, the nuget packages contains all libraries for the different systems. And at least on Linux and Windows I am able to run the examples without any problems. May I ask, if you are using a 32-Bit Linux system?

**Comment by darkguy2008** [comment]
> Hi! Yes, the nuget packages contains all libraries for the different systems. And at least on Linux and Windows I am able to run the examples without any problems. May I ask, if you are using a 32-Bit Linux system?

Hello! Thanks for your prompt response :)

Nope, I'm using 64-bit Kubuntu 20.04. While going around some other different libraries, I had to install some libraries that my OS seems to have missing. I'm thinking it could have been `libgtk-3-dev` and `libwebkit2gtk-4.0-dev`. I just retested and it seems to work. Assuming a line like this would fix it:
```
sudo apt install -y libwebkit2gtk-4.0-dev libgtk-3-dev
```
Maybe adding some documentation about extra packages to install if needed would be helpful :)
